java/server/src/org/openqa/selenium/grid/graphql/Node.java

```diff
@@ -33,18 +38,29 @@
   private final int maxSession;
   private final String capabilities;
   private static final Json JSON = new Json();
+  private final Set<Session> activeSessions;
 
 
   public Node(UUID id,
               URI uri,
               boolean isUp,
               int maxSession,
-              Map<Capabilities, Integer> capabilities) {
+              Map<Capabilities, Integer> capabilities,
+              Set<Session> activeSessions) {
     this.id = Require.nonNull("Node id", id);
     this.uri = Require.nonNull("Node uri", uri);
     this.isUp = isUp;
     this.maxSession = Require.nonNull("Node maxSession", maxSession);
     this.capabilities = Require.nonNull("Node capabilities", JSON.toJson(capabilities));
+    this.activeSessions = Require.nonNull("Active sessions", activeSessions);
+  }
+
+  public List<org.openqa.selenium.grid.graphql.Session> getSessions() {
```

Comment:
The Node should have no concept of graphql

Reply:
This is equivalent to `public List<Node> getNodes()` . Here the node is `org.openqa.selenium.grid.graphql.Node`. See: https://github.com/SeleniumHQ/selenium/blob/373159c40fbaab2c91074967b3da7e5c755f95d5/java/server/src/org/openqa/selenium/grid/graphql/Grid.java#L46
It was written this way because there are two classes with the name `Session`